pages/cockpit/user-guide/manage-organizations.adoc

```diff
@@ -0,0 +1,61 @@
+= Manage organizations
+:page-sidebar: cockpit_sidebar
+:page-permalink: cockpit/1.x/cockpit_userguide_manage_organizations.html
+:page-folder: cockpit/userguide
+:page-description: Gravitee.io Cockpit - Manage organization
+:page-keywords: Gravitee.io, API Platform, API Management, Cockpit, documentation, manual, guide
+
+== Overview
+
+Organizations in Cockpit correspond to APIM and AM organizations. Organization members and owners described in the procedures below can be users which are already defined with the appropriate role in APIM and AM.
+
+== What can you do with an organization?
+
+You can:
+
+* View details of the organization in the dashboard
+* Update the organization settings
+* Add members
+* Invite members
+* Transfer ownership of the organization
+
+You can access an organization from the *Organizations* menu on the left or by selecting the organization's dashboard icon image:icons\dashboard-icon.png[role="icon"] in the interactive map.
+
+== Update the organization settings
+
+. Click *Settings > General*.
+. Update the organization *Name*.
+. Update the organization *Description*.
+. Click *SAVE*.
+. Update the *Human Readable ID* and click *CHANGE HRID*.
+
+== Add a member to the organization
+
+You can add owner or user members to the organization.
+
+. Click *Settings > Members*.
+. Click the plus icon image:icons/cockpit-plus-icon.png[role="icon"].
+. Enter the member name. The name entered must match a registered member -- if it does not, click *INVITE A USER* and follow the instructions in the section below.
+. Select the role for the member.
+. Click the tick icon image:icons/enabled-icon.png[role="icon"] to save the user.
+
+== Invite a member to the organization
+
+You can invite users to become owner or user members of your organization.
+
+. Click *Settings > Invite a member*.
+. Enter the member email address.
+. Select the role for the member. If you are inviting the user for the purposes of transferring ownership of the organization, the role must be `ORGNIZATION_OWNER`.
+. Click *SEND INVITATION*.
++
+Cockpit sends the user an invitation to register a new user account.
```

Comment:
Same as in environment:
The user receiving the email will have to click on the link in the email to accept the invitation. He can either do it by creating a new account on Cockpit, or by logging into an existing one.